catalyst_voices_packages/catalyst_compression/catalyst_compression_web/assets/js/catalyst_compression.js

```diff
@@ -0,0 +1,73 @@
+const brotli = await import("https://unpkg.com/brotli-wasm@3.0.0/index.web.js?module").then(m => m.default);
+const zstd = await import("https://unpkg.com/@oneidentity/zstd-js@1.0.3/wasm/index.js?module");
```

Comment:
Are there any reasons why we decided not to pre-load these libraries?

Reply:
We load them as soon as you open the website so it will be available when you need it. We get it from npm's CDN 

What pre-loading you meant?

Reply:
Ah, I see, thanks.